apollo-sample/src/main/java/com/example/apollographql/sample/GitHuntApplication.java

```diff
@@ -0,0 +1,66 @@
+package com.example.apollographql.sample;
+
+import android.app.Application;
+
+import com.apollographql.apollo.ApolloCall;
+import com.apollographql.apollo.ApolloClient;
+import com.apollographql.apollo.cache.normalized.CacheKey;
+import com.apollographql.apollo.cache.normalized.CacheKeyResolver;
+import com.apollographql.apollo.cache.normalized.NormalizedCacheFactory;
+import com.apollographql.apollo.cache.normalized.lru.EvictionPolicy;
+import com.apollographql.apollo.cache.normalized.lru.LruNormalizedCacheFactory;
+import com.apollographql.apollo.cache.normalized.sql.ApolloSqlHelper;
+import com.apollographql.apollo.cache.normalized.sql.SqlNormalizedCacheFactory;
+
+import java.util.Map;
+
+import javax.annotation.Nonnull;
+
+import okhttp3.OkHttpClient;
+import okhttp3.logging.HttpLoggingInterceptor;
+
+public class GitHuntApplication extends Application {
+
+  private static final String BASE_URL = "https://githunt-api.herokuapp.com/graphql";
+  private ApolloClient apolloClient;
+
+  @Override public void onCreate() {
+    super.onCreate();
+    HttpLoggingInterceptor loggingInterceptor = new HttpLoggingInterceptor();
```

Comment:
is this necessary? It might make sense to keep the sample as simple as possible.